app/uk/gov/hmrc/apisubscriptionfields/service/FieldsDefinitionService.scala

```diff
@@ -41,6 +41,13 @@ class FieldsDefinitionService @Inject() (repository: FieldsDefinitionRepository)
     } yield fetch.map(asResponse)
   }
 
+  def getAll: Future[List[FieldsDefinitionResponse]] = {
+    Logger.debug(s"[getAll]")
+    for {
+      list <- repository.fetchAll()
```

Comment:
No need to use the for-comprehension in this case, as you have a single generator.
Just write:
```
repository.fetchAll().map(asResponse)
```